### PR TITLE
Update old photo

### PR DESCRIPTION
**Pull request by nazarbroslavskyi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
               </ul>
             </nav>
             <div class="introduce-myself"> 
-              <img class="my-photo" src="./img/46458592.jfif" height="148px" width="148px" alt="it is i))))">
+              <img class="my-photo" src="./img/nazar.jpg" height="148px" width="148px" alt="it is i))))">
```
Comment (Roka20012) [inline index.html]: change the alt
`it is i` -> `it's me` or `me`, `photo_of_myself`... something like that

You can read more about it here -> [why do we need appropriate alt for images](https://www.yola.com/blog/what-is-alt-text-and-why-do-your-photos-need-it/)

And also need to change the image photo name 
`nazar.jpg` -> `me.jpg` or something like that

**Review by Roka20012 — CHANGES_REQUESTED** [review]
Good work!
Few improvements and we are ready to go🏁

File: index.html (hunk 1 above)
Comment (nazarbroslavskyi) [reply index.html]: ok)

**Comment by Roka20012** [comment]
Nice work!

**Review by Roka20012 — APPROVED** [review]

**Pull request merged**